.circleci/test-deploy.yml

```diff
@@ -93,11 +93,11 @@ workflows:
   test-deploy:
     jobs:
       - integration-test-install:
-          name: integration-test-web-identity-profile
+          name: integration test web identity profile
```

Comment:
Instead of changing an existing test, it would be better to create a new one and name it in a way that conveys its purpose e.g. `test white space in job name` or something like that

Reply:
Good call, I added a whole separate test.